app-engine/java/src/com/google/cloud/tools/intellij/appengine/java/cloud/AppEngineDeploymentConfigurator.java

```diff
@@ -51,7 +51,7 @@
     deploymentSources.addAll(AppEngineUtil.createArtifactDeploymentSources(project));
     deploymentSources.addAll(AppEngineUtil.createModuleDeploymentSources(project));
 
-    if (PlatformUtils.isIdeaCommunity()) {
+    if (!PlatformUtils.isIdeaUltimate()) {
```

Comment:
This change is safe since we won't be exposing this module to other IDEs, it's only supported to work with IDEA via app-engine-java module, correct?

Reply:
actually, I don't know what I was thinking here. I think that I was anticipating the future where we expose this to other IDEs, but that doesn't make sense because these are Gradle sources and its Java only.

thanks for calling this out. let me change back and update the unit tests

Reply:
fixed